docs/getting_started/tests/test_flask.py

```diff
@@ -23,6 +23,7 @@
 
 
 class TestFlask(unittest.TestCase):
+    @unittest.skip("flaky test")
```

Comment:
should this be removed? i haven't seen this test fail outside of the wsgi import issues yesterday